<launch>
    <!-- MapServer 노드 실행 -->
    <node pkg="path_planning" type="MapServer.py" name="map_server" output="screen">
        <param name="map_file" value="$(find path_planning)/data/20250103_waypoint_graph.json" />
    </node>

    <!-- A-star_path 노드 실행 -->
    <node pkg="path_planning" type="A-star.py" name="a_star_path" output="screen">
        <param name="goal_tolerance" value="0.5" />
    </node>

    <!-- new_purepursuit 노드 실행 -->
    <node pkg="path_planning" type="PurePursuit.py" name="pure_pursuit" output="screen">
        <param name="lookahead_distance" value="5.0" />
    </node>
</launch>
